.gitignore

```diff
@@ -21,3 +21,5 @@ npm-debug.log*
 
 # examples
 examples/**/package-lock.json
+
+*.tgz
```

Comment:
what is the reasoning behind adding it here?

Reply:
I usually test local as packaged dependency (npm pack) (sometimes there are pitfalls from UMD/CommonJS that you see only when packaged). I put this to avoid push the *.tgz packed dependency. I can remove if you like it.

Reply:
Let's remove it as it's not really related to other changes in this PR